config/locales/devise/en.yml

```diff
@@ -5,22 +5,22 @@ en:
       already_confirmed: Your email address has already been confirmed. %{action}
       confirmed: You have confirmed your email address
       confirmed_but_must_set_password: You have confirmed your email address
-      send_instructions: >
-        You will receive an email with instructions for how to confirm your
-        email address in a few minutes.
-      send_paranoid_instructions: >
-        You will receive an email with instructions for how to confirm your email
-        address in a few minutes.
+      send_instructions: 'You will receive an email with instructions for how to confirm your email address in a few minutes.
+
+'
```

Comment:
This is an artifact of `i18n-tasks normalize` and our use of the `>`  when wrapping lines. The newlines were already there, this is just making them more explicit:

```ruby
require 'yaml'
=> true
YAML.load <<-STR
key:
   key: >
     some string
     that we wrap
   other: thing
STR
=> {"key"=>{"key"=>"some string that we wrap\n", "other"=>"thing"}}

```

I can make a pass and run `String#strip` on all our keys to remove these?